### DISCUSSION TITLE
Few ideas, comments welcome

### DISCUSSION THREAD

**Discussion by luni64** [luni64/EncoderTool#20] [Ideas]
Here a few ideas for future development. All comments or other ideas welcome
- Include some long/short button press logic to the Encoder classes
- Acceleration (i.e. increase count rate for fast rotation)
- Change resolution on the fly see [(here)](https://forum.pjrc.com/threads/70264-EncoderTool?p=306292&viewfull=1#post306292)
- Adding a `anyValueChanged()` for multiplexed encoders
- Support more boards

**Comment by Electrotechnique** [comment]
Acceleration is very useful. I implemented it by timing the interval between reads in `EncPlex74165::read()` and making extra callbacks. I have two levels of acceleration like this. Possibly not the best way.

**Comment by m-r-m-s** [comment]
Expanding on the discussion [here](https://github.com/luni64/EncoderTool/discussions/13), it would be great to have some long/short button press logic added to the Encoder classes. For example: short press, double click, and long press would be super helpful. Thank you.
